docs/list.md

```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Sign in with [Github here](https://devict-hacktoberfest.herokuapp.com/) so they can track your PRs.
 - **Notes**: You need to create at least two PRs in any of the  Wichita developer community organizations listed [here](https://devict-hacktoberfest.herokuapp.com/)
 
+#### **devtron**
+
+- **Swag**: Devtron's Limited Edition Swag Kit for 1st Rank , Devtron's Limited Edition Diary + stickers for 2nd Rank,  Devtron's Limited Edition stickers for 3rd Rank
+- **Requirements**:
```

Comment:
Please list the requirements of each event, summarized (do not just copy and paste from their Readme file...).